cmake/iwyu.cmake

```diff
@@ -7,7 +7,11 @@ if(NOT WIN32)
   ProcessorCount(CPU_COUNT)
 
   find_package(Python3 COMPONENTS Interpreter)
-  find_program(iwyu_tool_py NAMES iwyu_tool)
+  find_program(iwyu_tool_py NAMES iwyu_tool iwyu_tool.py)
+  set(fix_includes_py ${CMAKE_SOURCE_DIR}/third_party/include-what-you-use/fix_includes.py)
```

Comment:
Don't we usually use CAPS for CMake variables?

Reply:
Yes and no. :sweat_smile: Old-style cmake uses (all-caps) variables to transport information between scopes.
So you're always at risk of overwriting one of these variables. So I try to use lower case variables for all things local - if I  don't forget about it. So I'd rather stick to lower case here.

And yes it's not consistent. `CPU_COUNT` should be lower-case. I'm going to fix this.

Reply:
Okay, thanks for clarification. At least it is consistent now.